src/components/Trade/hooks/useTradeAmounts.ts

```diff
@@ -1,51 +1,220 @@
+import type { Asset } from '@shapeshiftoss/asset-service'
```

Comment:
i think the `.ts` file extension is causing poor syntax highlight on github diffs?

Reply:
not saying this is a problem per se just an observation

Reply:
You were right. It seems GitHub can't add syntax highlight a custom hook unless it has a `.tsx` extension.

Updated all related hooks in https://github.com/shapeshift/web/pull/2789/commits/19989cc4ab3014e99ace86fd4699b718eda293b6, which fixed the highlighting 👌